pkg/kubectl/cmd/exec/exec.go

```diff
@@ -40,27 +44,37 @@ import (
 
 var (
 	execExample = templates.Examples(i18n.T(`
-		# Get output from running 'date' from pod 123456-7890, using the first container by default
-		kubectl exec 123456-7890 date
+		# Get output from running 'date' command from pod mypod,
+		# using the first container by default
+		kubectl exec mypod date
 
-		# Get output from running 'date' in ruby-container from pod 123456-7890
-		kubectl exec 123456-7890 -c ruby-container date
+		# Get output from running 'date' command from first pod of the deployment named mydeployment,
+		# using the first container by default
+		kubectl exec deploy/mydeployment date
 
-		# Switch to raw terminal mode, sends stdin to 'bash' in ruby-container from pod 123456-7890
+		# Get output from running 'date' command from first pod of the service named myservice,
+		# using the first container by default
+		kubectl exec svc/myservice date
+
+		# Get output from running 'date' command in ruby-container from pod mypod
```

Comment:
Put this example next to the first one, to keep the consistency as before.

Reply:
updated